### ISSUE TITLE
Add vim-vdebug/vdebug for working with xdebug

### ISSUE THREAD

**Issue by salcode** [salcode/ironcode-vim#166]
Add [vim-vdebug/vdebug](https://github.com/vim-vdebug/vdebug) for working with xdebug based on instructions at https://webdevstudios.com/2019/04/16/debugging-wordpress-with-local-by-flywheel-neovim-and-xdebug/

Note: [vim-vdebug/vdebug](https://github.com/vim-vdebug/vdebug) seems to be the new location of [joonty/vdebug](https://github.com/joonty/vdebug), which stopped working with my Local by Flywheel setup and was removed (see #156)

I'm hoping to get this up and running once again with these fresh set of instructions and I'd prefer to use xdebug from within Vim rather than using another program.

**Comment by salcode** [comment]
After adding the `vim-vdebug/vdebug` plugin via VimPlug, I'm getting the error message

> :python3 is not available, vdebug will not be loaded.

When I run Neovim from the command line (`$ nvim`), running Vim 8.1 (`$ vim`) does not show this error. I am able to run vdebug successfully in Vim 8.1.

**Comment by salcode** [comment]
Based on some Googling, I came upon [these instructions on getting python3 running in Neovim](https://github.com/deoplete-plugins/deoplete-jedi/wiki/Setting-up-Python-for-Neovim#simple-setup) and I ran

```
pip3 install --user neovim
```

This fails

<details>

<summary>Failure to build greenlet - missing xcrun</summary>

```
  Failed building wheel for greenlet
  Running setup.py clean for greenlet
Failed to build greenlet
Installing collected packages: greenlet, pynvim, neovim
  Running setup.py install for greenlet ... error
    Complete output from command /usr/local/opt/python/bin/python3.7 -u -c "import setuptools, tokenize;__file__='/private/var/folders/rd/g5hl121d0mqf1jb0l611lyf00000gp/T/pip-install-oif9_h3m/greenlet/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /private/var/folders/rd/g5hl121d0mqf1jb0l611lyf00000gp/T/pip-record-7xzle941/install-record.txt --single-version-externally-managed --compile --user --prefix=:
    running install
    running build
    running build_ext
    building 'greenlet' extension
    creating build
    creating build/temp.macosx-10.11-x86_64-3.7
    clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -arch x86_64 -I/usr/local/include -I/usr/local/opt/openssl/include -I/usr/local/opt/sqlite/include -I/usr/local/Cellar/python/3.7.1/Frameworks/Python.framework/Versions/3.7/include/python3.7m -c greenlet.c -o build/temp.macosx-10.11-x86_64-3.7/greenlet.o
    xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
    error: command 'clang' failed with exit status 1

    ----------------------------------------
Command "/usr/local/opt/python/bin/python3.7 -u -c "import setuptools, tokenize;__file__='/private/var/folders/rd/g5hl121d0mqf1jb0l611lyf00000gp/T/pip-install-oif9_h3m/greenlet/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /private/var/folders/rd/g5hl121d0mqf1jb0l611lyf00000gp/T/pip-record-7xzle941/install-record.txt --single-version-externally-managed --compile --user --prefix=" failed with error code 1 in /private/var/folders/rd/g5hl121d0mqf1jb0l611lyf00000gp/T/pip-install-oif9_h3m/greenlet/
```
</details>

**Comment by salcode** [comment]
Running `xcrun` from the command line is throwing an error, which seems to be my next stop in correcting this.

```
$ xcrun
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
```

Based on [some Googling](https://apple.stackexchange.com/q/254380) I've run `xcode-select --install`, which seems to fix xcrun

```
$ xcode-select --install
xcode-select: note: install requested for command line developer tools
```

Now I can try running this again

```
$ pip3 install --user neovim
```

and now it works 👍 

and vdebug is now working in Neovim, as well 🎉

**Comment by salcode** [comment]
Resolved in eed6c4a2e1e012d5c658f0aaea159243051b311f
